pyrightconfig.stricter.json

```diff
@@ -59,6 +59,7 @@
         "stubs/peewee",
         "stubs/psutil",
         "stubs/psycopg2",
+        "stubs/pycocotools",
```

Comment:
This is automatically added by the stub generation script on the assumption that most contributed stubs will start off being somewhat incomplete. But I think you might have annotated everything, so I'm not sure it's actually needed!

```suggestion
```